src/EventStore.Core/TransactionLog/LogRecords/PrepareLogRecord.cs

```diff
@@ -55,6 +55,7 @@ public class PrepareLogRecord : LogRecord, IEquatable<PrepareLogRecord>, IPrepar
 		public string EventType { get; }
 		private int? _eventTypeSize;
 		public ReadOnlyMemory<byte> Data { get; }
+		private readonly ReadOnlyMemory<byte> _data;
```

Comment:
```suggestion
		private readonly ReadOnlyMemory<byte> _dataOnDisk;
```

suggestion: rename `_data` to `_dataOnDisk` to clearly distinguish between the two of them